src/oauth2/access_token/access_token.cr

```diff
@@ -54,7 +59,13 @@ abstract class OAuth2::AccessToken
   # will be the string "[1,2,3]".
   property extra : Hash(String, String)?
 
-  def initialize(@access_token : String, expires_in : Int?, @refresh_token : String? = nil, @scope : String? = nil, @extra = nil)
+  @[Deprecated("Use `token` instead of `access_token`")]
+  def initialize(access_token : String, expires_in : Int?, @refresh_token : String? = nil, @scope : String? = nil, @extra = nil)
```

Comment:
This doesn't work. The arguments are exactly the same as the next method. and so this will get overwritten and it wi. never show up in docs. Just delete this overload, there's no other way to do it.

Reply:
@asterite I removed this can you review this again?